test/apiv2/20-containers.at

```diff
@@ -31,12 +31,16 @@ t GET libpod/containers/json?all=true 200 \
   .[0].ExitCode=0 \
   .[0].IsInfra=false
 
-# Test compat API for Network Settings
+# Test compat API for Network Settings (.Network is N/A when rootless)
+network_expect=
+if root; then
+    network_expect='.[0].NetworkSettings.Networks.podman.NetworkID=podman'
+fi
 t GET /containers/json?all=true 200 \
   length=1 \
   .[0].Id~[0-9a-f]\\{64\\} \
   .[0].Image=$IMAGE \
-  .[0].NetworkSettings.Networks.podman.NetworkID=podman
+  $network_expect
```

Comment:
This is the one non-param-related change, the one I snuck in. All it does is, if running rootless, skip the `.NetworkSettings` check but run the others.